### PR TITLE
Add docker files

### PR DESCRIPTION
**Pull request by A5rocks**
Remade version of #31 because that one had a bugged amount of changed lines.

### CODE REVIEW

File: jolly-jellyfish/src/page_maker/models.py [hunk 1]
```diff
@@ -7,20 +7,11 @@
...
     name = models.CharField(max_length=80, unique=True, blank=False, null=False)
     date_created = models.DateTimeField(default=timezone.now)
     author = models.ForeignKey(User, on_delete=models.CASCADE)
-    # todo: option to upload own stylesheet rather than select from existing
-    # style_sheet = models.FileField(null=False, blank=False,
-    #                      upload_to='themes/', validators=[FileExtensionValidator(['css'])])
-
-    # match uses regex applied to base filename only(.*=any characters, \.=., $=end of string)
-    style_sheet = models.FilePathField(path=get_theme_path, recursive=True, match='.*\.css$')
+    style_sheet = models.FileField(null=False, blank=False, upload_to='styles/')
```
Comment (A5rocks) [inline jolly-jellyfish/src/page_maker/models.py:14]: Huh? What's this change for?

**Review by A5rocks — COMMENTED** [review]
Seems good though.

File: jolly-jellyfish/src/page_maker/models.py (hunk 1 above)
Comment (Artemis21) [reply jolly-jellyfish/src/page_maker/models.py:14]: Looks like it allows for uploading custom CSS files, since we have the built in ones in templates already anyway now.

File: jolly-jellyfish/src/config.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "secret_key": null,
+    "debug": true,
+    "allowed_hosts": []
+}
```
Comment (Artemis21) [inline jolly-jellyfish/src/config.json]: Is this file necessary? It will be generated if it does not exist.

File: jolly-jellyfish/src/initial-themes.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  fields:
+    name: galaxy
+    date_created: 2020-08-02 19:51:03.603472+00:00
+    author: 1
+    style_sheet: static/themes/galaxy.css
+- model: page_maker.template
+  pk: 5
+  fields:
+    name: win95
+    date_created: 2020-08-02 19:51:03.603472+00:00
```
Comment (Artemis21) [inline jolly-jellyfish/src/initial-themes.yaml]: Just a nitpick, but all these `date_created` fields seem pretty arbitrary... Perhaps we could set them to some epoch?
